### DISCUSSION TITLE
#!js -> interactive.fsharp and interactive.csharp missing; only have interactive.configureRequire

### DISCUSSION THREAD

**Discussion by EugeneRG1** [dotnet/interactive#2564] [Q&A]
I am trying to use the "getVariable" built in function from javascript to access variables defined in F#, but it appears that neither interactive.fsharp or interactive.csharp exist - and definitely the getVariable function doesn't either. The only thing that the interactive object seems to contain is the configureRequire function, Anyone else have this issue?
![image](https://user-images.githubusercontent.com/120351839/206953208-a8af6ecf-b0e5-4348-bc83-25b089103352.png)

**Comment by jonsequitur** [comment]
You can use `#!share` or `#!set` for this. JavaScript no longer has a different API for this.
